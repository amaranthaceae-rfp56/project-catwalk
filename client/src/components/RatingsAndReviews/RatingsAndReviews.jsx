import React from 'react';


const RatingsAndReviews = () => {
  return (
    <div data-testid = {'Ratings-And-Reviews'}>
      RATINGS AND REVIEWS
    </div>
  );
};

export default RatingsAndReviews;