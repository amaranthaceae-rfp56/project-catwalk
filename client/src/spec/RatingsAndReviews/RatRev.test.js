import {render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import RatingsAndReviews from '../../components/RatingsAndReviews/RatingsAndReviews.jsx';

afterEach( ()=> {
  cleanup();
});

test('test', () => {
  render(<RatingsAndReviews/>);
  const review = screen.getByTestId('Ratings-And-Reviews');

  expect(review).toBeInTheDocument();
  expect(review).toHaveTextContent('RATINGS AND REVIEWS');
});

test('snapshots', () => {
  const tree = renderer.create(<RatingsAndReviews/>).toJSON();
  console.log(tree);
  expect(tree).toMatchSnapshot();
});