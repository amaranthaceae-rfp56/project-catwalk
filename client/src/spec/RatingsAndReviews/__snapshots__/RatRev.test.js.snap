// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots 1`] = `
<div
  data-testid="Ratings-And-Reviews"
>
  RATINGS AND REVIEWS
</div>
`;
